packages/charts/chart-web-components/package.json

```diff
@@ -112,7 +108,8 @@
   "beachball": {
     "disallowedChangeTypes": [
       "major",
-      "minor"
+      "minor",
+      "patch"
```

Comment:
ran into this during https://github.com/microsoft/fluentui/pull/33647

creating a patch change file would be breaking change as it would bump to non pre-release version